tests/test_download_amex_transactions.py

```
@@ -0,0 +1,137 @@
+        args = parse_args(
+            ["--output-dir", "/tmp/amex", "--database", "parents_finance"]
+        )
+
+        self.assertEqual(args.output_dir, Path("/tmp/amex"))
+        self.assertEqual(args.database, "parents_finance")
+
+    def test_statement_months_are_parsed(self) -> None:
+        args = parse_args(
+            [
+                "--output-dir",
+                "/tmp/amex",
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp/amex"

(S108)